tensorfile.py

```diff
@@ -10,8 +9,8 @@
 file1.close()
 
 accuracy_list = []
-file_name2 = 'accuracy.txt'
-file2 = open(file_name2, 'r')
+file_name2 = "accuracy.txt"
+file2 = open(file_name2, "r")
```

Comment:
I recommend using the `while` statement whenever possible for opening and closing files to save yourself the trouble of 1) remembering to close the file when nothing goes wrong (which you have done) and 2) remembering to close the file when something goes wrong.

Reply:
`while` -> `with`?

Reply:
Yes, my mistake. I meant ``with``.